exotic/api/nea.py

```diff
@@ -128,21 +129,22 @@ def dataframe_to_jsonfile(dataframe, filename):
             f.write(json.dumps(jsondata['data'], indent=4))
 
     def _tap_query(self, base_url, query, dataframe=True):
-        # table access protocol query
-
-        # build url
-        uri_full = base_url
+        # Build the ADQL query string
+        adql_query = ''
         for k in query:
             if k != "format":
-                uri_full += f"{k} {query[k]} "
+                adql_query += f"{k} {query[k]} "
+        adql_query = adql_query.strip()  # Remove any trailing space
```

Comment:
This looks great. Only recommendation I have is making this a set comprehension, something like this:

```
adql_query = ' '.join(f"{k} {v}" for k, v in query.items() if k != "format")
adql_query = adql_query.strip()
```

I opened an issue so that I could also modify the way we create planet names and tagged this is part of it.

Reply:
Excellent suggestion @tamimfatahi  I made a new commit with the change.